general/code-peer-review-checklist.md

```diff
@@ -8,7 +8,7 @@ In order to consistently ensure high quality code, a peer review of all code cha
 2. Is the feature branch up-to-date with the `develop` branch?
 3. Does the continuous integration build (e.g. Travis, CircleCI) pass? Do any of the non-required continuous integration steps fail? If so, why? Can these be fixed? If not, why not?
 4. If any external dependencies were added:
-    1. Review the results of the [Tidepool External Dependency Considerations] for each new dependency and sub-dependency.
+    1. Review the results of the [Tidepool External Dependency Considerations](https://github.com/tidepool-org/docs/blob/master/general/external-code-dependency-considerations.md) for each new dependency and sub-dependency.
```

Comment:
This is actually using a Markdown reference syntax; the actual (broken) link is at the bottom of this same MD file.